insights/src/main/kotlin/com/algolia/instantsearch/insights/webservice/Extensions.kt

```diff
@@ -45,3 +47,12 @@ internal fun WebService.uploadEvents(database: Database, indexName: String): Lis
     database.overwrite(failedEvents.map { it.event })
     return failedEvents
 }
+
+
+internal fun JSONArray.toList(): List<Any> {
+    return mutableListOf<Any>().also {
+        for (i in 0..it.size) {
```

Comment:
```suggestion
        for (i in 0 until it.size) {
```
If we have a size of 4, the last index will be 3, so we want to use `until` instead of `..` I think, unless I'm missing something

Reply:
:man_facepalming: